###learning-resource

##learning

#Guide to statistical analysis in microbial ecology -> mb3is.megx.net/gustame

#An Introduction to Applied Bioinformatic -> http://readiab.org/

#Gut Check: Exploring Your Microbiome (Coursera) -> https://www.coursera.org/learn/microbiome

##bioinformatics:tutorial and practice

#Useful sites for beginner
http://www.metagenomics.wiki/pdf/definition -> introduction and examples of tools
http://manual.omicsbox.biobam.com/user-manual/module-metagenomics/ -> introduction of tools
https://astrobiomike.github.io/genomics/ -> introduction and tutorials of tools
http://merenlab.org/tutorials/assembly-based-metagenomics/ -> tutorial on overview of assembly-based metagenomics

#EBI online training tutorials
https://www.ebi.ac.uk/training/online/topic/metagenomics

#Tutorials from USDS metagenomics workshops
https://usda-ars-gbru.github.io/Microbiome-workshop/tutorials/metagenomics/ -> tutorial on preprocessing and assembly with example data set
https://usda-ars-gbru.github.io/Microbiome-workshop/tutorials/metagenomics2/ -> tutorial on annotation and binning


#Happy Belly Bioinformatics -> https://astrobiomike.github.io/

#Metagenomics wiki by Matthias Scholz -> http://www.metagenomics.wiki/
download genome from ncbi ftp->http://www.metagenomics.wiki/tools/fastq/ncbi-ftp-genome-download

#Genomes Download (FTP) FAQ -> https://www.ncbi.nlm.nih.gov/genome/doc/ftpfaq/#howtofind

##bioinformatics:basics& magics for linux & slurm

#Ijaz's Bioinformatics tutorials and scripts -> http://userweb.eng.gla.ac.uk/umer.ijaz/bioinformatics/?C=M;O=D
#Ijaz's Linux command line exercises for NGS data processing -> http://userweb.eng.gla.ac.uk/umer.ijaz/bioinformatics/linux.html

#Batch jobs and job scripting -> https://curc.readthedocs.io/en/latest/running-jobs/batch-jobs.html

#Slurm Quick Start User Guide -> https://slurm.schedmd.com/quickstart.html

#useful tricks for processing multiple files in shell script -> https://otagomohio.github.io/hackyhour/sessions/2019_09_11.html
